package.json

```diff
@@ -2,20 +2,19 @@
   "name": "mmmines",
   "private": true,
   "scripts": {
-    "dev": "esrun app/server.ts",
-    "build": "next build",
+    "dev": "esrun --watch app/server.ts",
```

Comment:
it would be nice to run concurrently with `fly redis connect` but I don't think that would be possible unless we use the redis-cli directly

Reply:
yeah I tried this, but the fly cli command for connecting to redis cannot be run in non-interactive mode. 🙃